dev/ssl-stores/client-test-keypair.pem

```diff
@@ -0,0 +1,89 @@
+it/uooHWtMEQNjQN0d9WQ7oUWh1KwoK98Wb+WOPeKioUZYYEhyS3BZzY1yPNuHkg
+F21Sja95XeEVmSURNb6ckYzkP0j3kNnyIEIvTuR4OV0Ivn7KfxFgXpcg29BQtJcg
+i+58LN7t46W6NV1gic5DzBUFtYJ7ekitIyki+EJPo90slnSSHVwqwsXBE+ZM476s
+LG6vyaXEJJId2rnjxbjZ8RBs7oSV16KsIUcNOo32GJgrZQZFZlwZO20JBfXF/zYQ
+rs+K77QMYpLHqitT9lJ8kBhMUNnapP9tBjMbqsZCRQKCAQBFLdLYj7M+3KX4mcqL
+V5JkpnQKRy6/d+7aaBZx2/YEtfMAOgObUooulNVAp5ARNC2hVeWqYKpay1qkoc7k
+E6rAKJ2tM2ZkQpOGoubSom4AENX+MH8+SxIswI6XVQegawHJXqhR2y4XvK6EcyZG
+v1kBMbN8bJSanxfTUduhiMg/Gh0BW7v+ajQ3AMXPakFq6VlYUUVhJJYE2vG00NTP
+DcNPLukoEXARE9lPP5Gu902zRz4cLeMDvmnVTFSUq6pSmJ+z+IAHH7As+5XgQ2lL
+RRG0FLwpa7ijEFdXXr1IKsjuabgizS8LXKEV/VFqk6A1V/wgy75kQXG6X+Oy1GTY
+geSxAoIBAEyvlCZ7I9h+KxqHQPSOyWtFxmTaTItt4LaIJScSjMEBGVp8tfjIwq7V
+3ImTf4JsU0LvAT67fB/a1I7A5qRMokhVE/h91EKtSslwQWCM7SEjzGp6WDZ5u6JJ
+7Qd4arOVaHQikYEXkOchggnB7DX+nSBx1LX19l6UvaUlK25jpB8ShdaiSh7pBx/R
+VKivQRft6VgE+5K++yMx34TYyyuwWdq4xY2pveOyKz9XBhK1sQeUF5IWCeob3A0S
+vUAJbn0NtSvVJvYA8GqvirejQ+N0mdwJ77lFsFuKA5tpcBsljzN4fMMD7vPnxIUF
+Pco7CkbyheXsBq6xLWBaS5MxtHArz+I=
+-----END PRIVATE KEY-----
+Bag Attributes
+    friendlyName: client-test
+    localKeyID: 54 69 6D 65 20 31 35 30 34 39 31 33 33 30 35 35 37 37 
+subject=/CN=client-test
+issuer=/CN=client-test
+-----BEGIN CERTIFICATE-----
+MIIEyzCCArOgAwIBAgIERMCATDANBgkqhkiG9w0BAQsFADAWMRQwEgYDVQQDEwtj
+bGllbnQtdGVzdDAeFw0xNzA5MDgyMzI4MjVaFw0yNzA5MDYyMzI4MjVaMBYxFDAS
+BgNVBAMTC2NsaWVudC10ZXN0MIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKC
+AgEAmFbFaTdyrQb7YsQBAaHhXClgX9q7G/qdk/W2cybvNpizEOeoBZTUNoFRH0y+
+KK3LMcLEXUc9/PZVDmDK0IqnXqVK2091ukdciiDidbuTouqDCDQwWLb5zdbnIPgj
+UX7E+AKaon1K4VuY4o/7ezPGm6BVrX2Qer6phECGNq6gsqEqEKRq0HOrmC5pPTo3
+smctYPhE0DX1WwTiGz9Ym/uMkmdcx1nPswHlWpk9wDnR7iVlkLhm4KqUl+a1+4VP
+HGBxu1Ya746MkHU3QTEWpYWwXOVJsauRdq04GPxKkv+haJ6glJI3K08DPZOWi+H6
+QsebeCZVETbSDLohlstyt8hg2mlpoYmGgQnrJyiwu61SOhVpAQT9fTDcD6LWkzIX
+yAGoOfofHs19nZJriffoLK04fDfwc/nfF8aEZl8EkjOVKIwf1YgIETXmm0jQgKn9
+0nnXX/DWBESMkeHeQk+Vcty/OFh2lkY5Wdp3mdc+vMwmGa7H5JxIIFG9BCpnljKF
+R2EIMdX3lnds3jz35Uv0M7/cGLzVo1R+Bf8TmFOkLJVpOnxxoWaRNX2o5RTJ83wn
+l4sk0e5G3SHqrdaFVHVp7dR1RQ0H6ctFFqheA4WaOCqzTrzL5DN4myEkuX2RPbBq
+nvenh2WZijqWRvHgnxKTMkTipCEfvgdP580Smvle45ss9NcCAwEAAaMhMB8wHQYD
+VR0OBBYEFIEeSEr+uhKgNuE7aFZVjHywtCbHMA0GCSqGSIb3DQEBCwUAA4ICAQCS
+kmStKe3UOOXJPLMyFN4HaqyKE1EY8XxpQ0aJs04BpuLlyx4p3DE+3sKKAaRXKxX6
+3BYMXHdQmFzncJ79TM+8DJtv0w8Nu9au2JXgBmHFgCt2U56JXUQ/pBC3HGC1CJT0
+J0G2thQccUXS1If3kcD6S9bv0uqI8INFoOZKzRXYxCXEuOF9E+E4NOqzwqBqnbVz
+9WyT6wHUs/NPVbSD7MQcXud25CZ7pMhV29pJWjSIJdK5r+/H/9PfeGthaBjoiSR5
+g0edNdsVybDtrLPq8wUYehWYnt/RbS1l6OYPzeuzh93sUerZ3dTHfFsilCaVvnod
+nkregDKVW6jk/sMd8KeIGrdDSy5+AAoeU5ORKG4Sk168v0j609HbKrJyu77wYqo3
+GxYfI2MRPcerCgqpROXpSgDrmAP9ongBlyTrhimvTa5glzZjmSFbhbrQXgWzGgnR
+XugoDGtjNDCy4SfI7QLQKWvqXcxZbDACFXPeGFA8TvFBFTYxeC5r3Dh2CsFgt/pP
+VgKcUVQICZI7onzZHq+zpKpaPd/s/evyR8Mau8jhSHtLPlR3RcTdZOIDnWs3HEdJ
+26gDAiX0BerZghS4KuHPTRRhYAJWiV3BN6CVWNvkQAJFYOeY6zLPspxtu2JZbWVo
+waUfnyPo4LKjCl6vrf1afbvIbJuzLribuNi7zfWp6g==
+-----END CERTIFICATE-----
```

Comment:
Please remove the `.DS_Store` file(s) and add a `.gitignore` entry for them.

(Commenting here because I can't on it.)

Reply:
Done.